### PR TITLE
Added ability to load .env file from resolution path.

### PR DESCRIPTION
**Pull request by Swoorup**
* Merges env variables from the .env file automatiblity
* If not found it searches up to 3 levels up from the resolution path.

Unfortunately I had to remove .net45 due to package incompatibility.

### CODE REVIEW

File: src/FSharp.Data.LiteralProviders/Env.fs [hunk 1]
```diff
@@ -2,7 +2,28 @@
...
+    let envFile =
+        seq {
+            let mutable currentDirectory = DirectoryInfo(baseDir)
+            
+            for _ in [1..LevelsToSearchForEnvFile] do
+                yield! currentDirectory.EnumerateFiles(DefaultEnvFileName, SearchOption.TopDirectoryOnly)
+                currentDirectory <- currentDirectory.Parent
+        }
+        |> Seq.map (fun f -> f.FullName)
+        |> Seq.tryHead
```
[the remark this answers is not in the document]
Comment (Swoorup) [reply src/FSharp.Data.LiteralProviders/Env.fs:22]: Usually env files are pretty barebones, I don't think it should load them all as they usually are not organized hierarchically to my knowledge. As you mentioned this only picks the first closest match and stops, and if not count searches in the parent directory and repeats for max 3 times. 

I borrowed the autoloading functionality from here.
https://github.com/bolorundurowb/dotenv.net/blob/master/src/dotenv.net/DotEnv.cs#L59

I think something useful would probably be to specify loading .env file optionally in the parameter.

File: src/FSharp.Data.LiteralProviders/paket.template [hunk 2]
```diff
@@ -1,7 +1,9 @@
-﻿type project
+type project
 files
   ..\..\packages\tpsdk\NETStandard.Library.NETFramework\build\net461\lib\netstandard.dll ==> lib/netstandard2.0
   ..\..\packages\tpsdk\NETStandard.Library.NETFramework\build\net461\lib\System.Reflection.dll ==> lib/netstandard2.0
   ..\..\packages\tpsdk\NETStandard.Library.NETFramework\build\net461\lib\System.Runtime.dll ==> lib/netstandard2.0
+  bin/Release/netstandard2.0/*.dll ==> lib/netstandard2.0
```
[the remark this answers is not in the document]
Comment (Swoorup) [reply src/FSharp.Data.LiteralProviders/paket.template:6]: Will do. 

I really wish there was some automated way to include just the required dependencies :/

File: src/FSharp.Data.LiteralProviders/Env.fs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Swoorup) [reply src/FSharp.Data.LiteralProviders/Env.fs:22]: @Tarmil 
Maybe its beneficial such that `.env` files are always loaded, and you could have `.env.development` on top which are manually specified and loads on top of already loading existing envs which could be a future PR.

I know react does something similar. https://create-react-app.dev/docs/adding-custom-environment-variables/

image

[the remark this answers is not in the document]
Comment (Tarmil) [reply src/FSharp.Data.LiteralProviders/Env.fs:22]: Okay, this is fine then 🙂

Agreed about parameterizing the file name, this would be nice.

**Comment by Swoorup** [comment]
Sorry, @Tarmil a github bot automatically wiped out all my changes in master. Do you want me to create another pull request. Might have changes locally.

**Comment by Tarmil** [comment]
No worries, I found the changes on your `standard-only` branch. I adapted them on the project structure refactor I made on master: #8

**Pull request merged**